src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -369,6 +374,9 @@ class GrpcLb : public LoadBalancingPolicy {
 
   // Who the client is trying to communicate with.
   const char* server_name_ = nullptr;
+  // The target name from configuration; if set, it overrides server_name_ in
+  // the balancer requests.
+  std::string target_name_;
```

Comment:
Instead of storing this and `child_policy_config_` separately, I suggest just storing a ref to the `GrpcLbConfig` object.  Then we can read the fields out of it as needed.